integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/exception/WebClientWithMessageException.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.openhouse.javaclient.exception;
+
+import org.springframework.web.reactive.function.client.WebClientException;
+
+/**
+ * An exception thrown in Openhouse clients to indicate an error in a tables API response or
+ * request, acting as a wrapper around WebClientException.
+ */
+public abstract class WebClientWithMessageException extends RuntimeException {
+  private final WebClientException exception;
+
+  public WebClientWithMessageException(WebClientException exception) {
+    super(exception);
+    this.exception = exception;
+  }
+
+  public String getMessage() {
```

Comment:
What is usage of this function?

Reply:
we actually don't need this either. removed in latest commit